### DISCUSSION TITLE
Download web page and convert to EPUB?

### DISCUSSION THREAD

**Discussion by Shohreh** [jgm/pandoc#10283] [Q&A] [answered]
Hello,

I thought it'd be a breeze to download a web page and turn it into an EPUB file, either with just pandoc or with wget + pandoc… but it doesn't work: In one page in the EPUB, I get a mysterious "Couldn't render this page".

`pandoc.exe --metadata title="My title" -o "My file.epub" https://www.acme.comblah.html`

Is there a simple way to get pandoc to turn a web page into an EPUB file?

Thank you.

**Comment by jgm** [comment, marked as answer]
There's no way to answer this question without knowing what is going on with that page.
The URL you give is a 404 so I couldn't tell.
But in general you should be able to do something like this.
Just tried it with `pandoc https://pandoc.org -o pandoc.epub` and it worked just fine.

**Reply by Shohreh** [reply]
Thanks for the confirmation. I'll investigate why the specific page shows an error.
